<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1549988912167' stop='1549988934461'>
    <name>workspace-project App</name>
    <title>workspace-project App</title>
    <test-cases>
        <test-case start='1549988912168' status='passed' stop='1549988932047'>
            <name>Test Initial Loading Site</name>
            <title>Test Initial Loading Site</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='cccb12c8-8799-454b-8b11-2dbd11266af4-attachment.png' type='image/png' size='66383'/>
            </attachments>
        </test-case>
        <test-case start='1549988932048' status='failed' stop='1549988934460'>
            <name>Test Second Loading Site</name>
            <title>Test Second Loading Site</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='a2f18fb6-9928-4be5-8812-f7c789542196-attachment.png' type='image/png' size='69150'/>
            </attachments>
            <failure>
                <message>Failed: A test is already running</message>
                <stack-trace>Error: A test is already running
                    at Eyes.&lt;anonymous> (/Users/jbarrera/Protractor/HugeAutomation/node_modules/eyes.sdk/src/EyesBase.js:918:32)
                    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)
                    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
                    at processTicksAndRejections (internal/process/next_tick.js:81:5)
                From: Task: &lt;anonymous>
                    at PromiseFactory._promiseFactoryFunc (/Users/jbarrera/Protractor/HugeAutomation/node_modules/eyes.selenium/src/Eyes.js:76:25)
                    at PromiseFactory.makePromise (/Users/jbarrera/Protractor/HugeAutomation/node_modules/eyes.utils/src/PromiseFactory.js:38:25)
                    at EyesBase.open (/Users/jbarrera/Protractor/HugeAutomation/node_modules/eyes.sdk/src/EyesBase.js:897:37)
                    at /Users/jbarrera/Protractor/HugeAutomation/node_modules/eyes.selenium/src/Eyes.js:179:44
                    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)
                    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
                    at processTicksAndRejections (internal/process/next_tick.js:81:5)
                From: Task: Run beforeEach in control flow
                    at UserContext.&lt;anonymous> (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:94:19)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at QueueRunner.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:909:35)
                    at Spec.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:526:10)
                    at UserContext.fn (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:5340:37)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at runNext (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (/Users/jbarrera/Protractor/HugeAutomation/e2e/app.e2e-spec.ts:62:3)
                    at addSpecsToSuite (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
                    at Env.describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
                    at describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
                    at Object.&lt;anonymous> (/Users/jbarrera/Protractor/HugeAutomation/e2e/app.e2e-spec.ts:57:1)
                    at Module._compile (internal/modules/cjs/loader.js:734:30)
                    at Module.m._compile (/Users/jbarrera/Protractor/HugeAutomation/node_modules/ts-node/src/index.ts:439:23)
                    at Module._extensions..js (internal/modules/cjs/loader.js:745:10)
                    at Object.require.extensions.(anonymous function) [as .ts] (/Users/jbarrera/Protractor/HugeAutomation/node_modules/ts-node/src/index.ts:442:12)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>